docs/source/references/config.md

```diff
@@ -10,7 +10,7 @@ The following is the default configuration used by Iroha.
 {
   "PUBLIC_KEY": "ed0100",
```

Comment:
Not sure if it's the right time, but we should have a better placeholder.

Reply:
not sure that I get you, this is multihash of empty public key. I'd argue that we need a better defaults for public-private key pair

Reply:
I agree, we need better defaults.

Reply:
I'll open an issue to discuss this